lib/views/submit.js

```diff
@@ -349,7 +349,7 @@ function submitPost (req, res) {
           }
           res.send(pug.renderFile('templates/views/errors/errors.jade', locals))
         }
-      })
+      }) */
```

Comment:
How come this is commented out?

Reply:
It seems to be causing an unhandled promise rejection.  Actually, I'm still seeing unhandled promise rejections when you try to merge and are prevented due to trying to submit non-identical objects into an existing collection, but this seemed to move it closer to where it was happening.  It is strange. Still bad at promises

Reply:
Refactoring to use async/await should make it much more clear where exceptions are going.  Will accept to solve the immediate issue but we need to sort this soon.